src/sparql/reports/existingcomment.sparql

```diff
@@ -0,0 +1,20 @@
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix MONDO: <http://purl.obolibrary.org/obo/MONDO_>
+
+#description: terms with comments
+
+SELECT DISTINCT ?entity ?comment WHERE {
```

Comment:
Maybe it's too early for me to review; I see that this is just in draft!

I'm assuming that this will change to a sparql update, and will be run on the 10 or so terms in the spreadsheet!

Reply:
yes, it is still early.
I think the reason you were added is because this PR will include a Sparql query for report. 
I'll change this PR into "ready to review" when I have added tags

Reply:
@sabrinatoro is there an updated query to commit?

Reply:
Or was this file not intended to be committed?

Reply:
@sabrinatoro I know this is is merged now, but just wanted to follow up. Was this file intentionally added? Is it related to the main work in this PR or is it of a different purpose?

I just mention that because I expected maybe this file was boilerplate and it was going to change to an update query that updated the ontology to add the obsoletion candidate related edits, but that doesn't seem to be teh case.

Reply:
@joeflack4 great question! I had a chance to chat with Sabrina earlier today and this is an example query so curators can run this in general, modulo adding in new curies, when this needs to be done in the future.